Firestore/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 - [changed] Firestore no longer has a direct dependency on FirebaseAuth.
 - [changed] Removed the includeMetadataChanges property in FIRDocumentListenOptions
   to avoid confusion with the factory method of the same name.
+- [changed] FIRWriteBatch method commitWithCompletion has nullable param and a new
```

Comment:
This could be rephrased:

```
Added a commit method that takes no completion handler to FIRWriteBatch.
```

We don't need to call out that commitWithCompletion is now nullable. We did this because we could and for simplicity but the intended API is just to call commit.

Reply:
done